src/Language/en/Auth.php

```diff
@@ -25,7 +25,7 @@
     'badToken'              => 'The access token is invalid.',
     'oldToken'              => 'The access token has expired.',
     'noUserEntity'          => 'User Entity must be provided for password validation.',
-    'invalidEmail'          => 'Unable to verify the email address matches the email on record.',
+    'invalidEmail'          => 'Unable to verify the email address matches the email {0}.',
```

Comment:
Is this better?
```suggestion
    'invalidEmail'          => 'Unable to verify the email address matches the email "{0}".',
```

![Screenshot 2024-07-19 11 26 08](https://github.com/user-attachments/assets/750b1357-dc18-40e4-ad66-d31033f0bbf2)

Reply:
ahh, that's much better. I didn't even think about it.